cocos/root.jsb.ts

```diff
@@ -115,7 +115,6 @@ rootProto._ctor = function (device: Device) {
     this._batcher = null;
     this._pipelineEvent = new DummyPipelineEvent();
     this._debugView = new DebugView();
```

Comment:
No need to new the `web version` DebugView here. Native DebugView has been bound.